### DISCUSSION TITLE
Hyde Discord 🔗 Join us!

### DISCUSSION THREAD

**Discussion by kRHYME7** [prasanthrangan/hyprdots#1235] [Show and tell]
🎉 Hello Everyone! Exciting news: Hyprdots is evolving into HyDE (still Hyprdots, but with ~awesome~ hypr upgrades)! 🎉

🔖 Here's what you can expect:

 * Tagged releases
 * Significant improvements and updates
 * Major implementation enhancements

Yep, that's the gist of it!

Since these changes might shake things up a bit, we're launching a Discord server. 😁 It's a place where you can share your thoughts and ideas as a community.

🔗 Join us here: [Discord Invite Link](https://discord.gg/qWehcFJxPa) (The server is incomplete and will be glad if you can provide some inputs to make the server even Hypr)

That's all for now! See you there! 🚀

**Comment by drnikos** [comment]
Could you update the invite link because it is invalid or it has expired?

**Reply by kRHYME7** [reply]
Yes done!

**Comment by FlipperPlz** [comment]
Damn didnt realize the person who created my dotfiles was a raging trasphobe. Crazy that your dotfiles are platformed by hyprland's wiki but am sure that will change when more people look into the comments in your discord.

**Reply by FlipperPlz** [reply]
![Screenshot_20250510_110645_Discord](https://github.com/user-attachments/assets/0b66d894-5787-4471-a6e4-2f668dc01307)
![Screenshot_20250510_110624_Discord](https://github.com/user-attachments/assets/6c7ccd30-68ff-4e2c-9545-d9660b760d72)

**Reply by kRHYME7** [reply]
>  Damn didnt realize the person who created my dotfiles was a raging trasphobe. 

Damn no part of the commit history,his PR being merged so please spare this repo of whatever things you're thinking.Let me know I will revert it!

  I humbly request.  Let's just make these dotfiles 😎 cool.

I am the one maintaining it and please stop, respectfully.

**Reply by FlipperPlz** [reply]
Idk just seems weird to advertise a community thats free of hatespeech when theres already a history of that by your own staff team. I would issue some punishments or revert the discord. Keeping a good reputation isn't hard, the people on your staffteam reflect you and your work.

**Reply by kRHYME7** [reply]
> Idk just seems weird to advertise a community thats free of hatespeech when theres already a history of that by your own staff team.

Thank you for clarifying about being it a history. I am happy to tell you that it was already resolve way back then.  

 >I would issue some punishments or revert the discord. 

Appreciated this one. 


> Keeping a good reputation isn't hard, 

I agree that maintaining a good reputation isn’t difficult-it's only challenging when someone intentionally tries to damage it.

> the people on your staffteam reflect you and your work.

As for the comment that the staff team reflects me and my work, I respectfully disagree. While the team is important, I have a large circle of friends and colleagues, and sometimes mistakes happen. However, those incidents have been addressed and resolved. Please don’t attribute those past issues directly to me or to the project. 

---

Everyone is free to contribute to this dot files, and I will be grateful to review it.

---

For anyone following this thread, feel free to DM me when discord chat feels uncomfortable.  I can't cater everyone, but at least I want everyone to be friends at least! Cool?


#  I love you all!

**Comment by w0lzard** [comment]
I am unable to join the server it says the link has expired
